docs/user-guide/pod-security-policy/index.md

```diff
@@ -39,7 +39,7 @@ into three categories:
 restrictive value. 
 - *Controlled by an allowable set*: Fields of this type are checked 
 against the set to ensure their value is allowed.
-- *Controlled by a strategy*: Items that have a strategy to generate a value provide
+- *Controlled by a strategy*: Items that have a strategy to provide
```

Comment:
@pweil- I've added another improvement ^^^. Let me know If I should squash commits.